components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -54,145 +57,202 @@ const TASKS = {
+            }
+            const result = await getGitpodService().server.triggerPrebuild(project.id, null);
+            setStartPrebuildResult(result);
+        } catch (error) {
+            setPrebuildWasTriggered(false);
+            setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error).replace(/Error: Request \w+ failed with message: /, '')} />);
+        }
     }
-    try {
-      setPrebuildWasTriggered(true);
-      if (!isEditorDisabled) {
-        await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
-      }
-      const result = await getGitpodService().server.triggerPrebuild(project.id, null);
-      setStartPrebuildResult(result);
-    } catch (error) {
-      setPrebuildWasTriggered(false);
-      setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error).replace(/Error: Request \w+ failed with message: /, '')}/>);
+
+    const onInstanceUpdate = (instance: WorkspaceInstance) => {
+        setPrebuildInstance(instance);
     }
-  }
 
-  const onInstanceUpdate = (instance: WorkspaceInstance) => {
-    setPrebuildInstance(instance);
-  }
+    useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
 
-  useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
+    return <>
+        <Header title="Configuration" subtitle="View and edit project configuration." />
+        <div className="lg:px-28 px-10 mt-8 flex space-x-4">
+            <div className="flex-1 h-96 rounded-xl overflow-hidden relative flex flex-col">
+                <div className="flex bg-gray-50 dark:bg-gray-800 border-b border-gray-200 dark:border-gray-600 px-6 pt-3">
+                    <TabMenuItem name=".gitpod.yml" selected={selectedEditor === '.gitpod.yml'} onClick={() => setSelectedEditor('.gitpod.yml')} />
+                    {!!dockerfile && <TabMenuItem name=".gitpod.Dockerfile" selected={selectedEditor === '.gitpod.Dockerfile'} onClick={() => setSelectedEditor('.gitpod.Dockerfile')} />}
+                </div>
+                {editorMessage}
+                <Suspense fallback={<div />}>
+                    {selectedEditor === '.gitpod.yml' &&
+                        <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
+                    {selectedEditor === '.gitpod.Dockerfile' &&
+                        <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="dockerfile" value={dockerfile} onChange={setDockerfile} />}
+                </Suspense>
+                {isDetecting && <div className="absolute h-full w-full bg-gray-100 dark:bg-gray-700 flex items-center justify-center space-x-2">
+                    {showAuthBanner ? (
+                        <div className="mt-8 text-gray-500 flex-col">
+                            <div className="p-6 text-center">
```

Comment:
**nitpick**: Minor spacing issue. What do you think? It seems this will also _not_ affect this element in other places like project configuration.

```suggestion
                            <div className="p-16 text-center">
```

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2021-09-29 at 2 08 26 PM (2)" src="https://user-images.githubusercontent.com/120486/135257044-846130bb-1f7e-479d-b309-c353d45aee81.png"> | <img width="1440" alt="Screenshot 2021-09-29 at 2 09 43 PM (2)" src="https://user-images.githubusercontent.com/120486/135257218-f42e7e17-998f-45b3-973c-09c8b42cff4b.png"> |